README.md

```diff
@@ -1,19 +1,31 @@
-# Akubra [![Build Status](https://travis-ci.org/allegro/akubra.svg?branch=master)](https://travis-ci.org/allegro/akubra) #
-## Project goal ##
+# Akubra
+[![Version Widget]][Version] [![Build Status Widget]][Build Status] [![GoDoc Widget]][GoDoc]
 
-Akubra is a simple solution to keep independent S3 storages in sync eventually (almost realtime).
+[Version]: https://github.com/allegro/akubra/releases/latest
+[Version Widget]: https://img.shields.io/github/release/allegro/akubra.svg
+[Build Status]: https://travis-ci.org/allegro/akubra
+[Build Status Widget]: https://travis-ci.org/allegro/akubra.svg?branch=master
+[GoDoc]: https://godoc.org/github.com/allegro/akubra
+[GoDoc Widget]: https://godoc.org/github.com/allegro/akubra?status.svg
 
-Keeping redundant storage clusters which handle great volume of new objects (~300k/h), is most
-effective by feeding them with all data incoming at once. That's what Akubra does, with minimum
-memory and cpu footprint.
+## Goal
 
-Syncronizing S3 storages offline is almost impossible with high volume traffic. It would require
-keeping track of new objects (or periodical buckets listing...), downloading and uploading them to
-other storage. It's slow, expensive and hard to implement.
+Akubra is a simple solution to keep a independent S3 storages in sync - almost
+realtime, eventually consistent.
 
-Akubra way is to put files in all storages at once by copying requests to multiple backends. Sometimes
-one of clusters may reject request for various reason, but that's not a big deal: we simply log
-that event, and sync object in the independent process.
+Keeping redundant storage clusters, which handle great volume of new objects
+(about 300k obj/h), is the most efficient by feeding them with all incoming data
+at once. That's what Akubra does, with a minimum memory and cpu footprint.
+
+Synchronizing S3 storages offline is almost impossible with a high volume traffic.
+It would require keeping track of new objects (or periodical buckets listing),
+downloading and uploading them to other storage. It's slow, expensive and hard
+to implement.
+
+Akubra way is to put files in all storages at once by copying requests to multiple
+backends. Sometimes one of clusters may reject request for various reason, but
+that's not a big deal: we simply log that event, and sync object in the
```

Comment:
"sync that object" or "sync objects" plus "in an independent process" (first mentioned here).